app/src/main/java/com/stacker/simplegithubapp/FollowerActivity.kt

```diff
@@ -0,0 +1,17 @@
+package com.stacker.simplegithubapp
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+
+class FollowerActivity : Fragment() {
```

Comment:
Activity와 Fragment는 다릅니다.
이름은 Activity로 되어잇는데 상속은 Fragment를 받잖음?
Activity가 아님. 

```suggestion
class FollowerFragment : Fragment() {
```